launch.py

```diff
@@ -19,12 +22,21 @@ def env_defined(key):
 
 
 def random_passphrase():
-    password = "'"
-    while "'" in password:
-        with open("/usr/share/dict/american-english") as f:
-            words = f.readlines()
-        password = "-".join(random.sample(words, 2)).replace("\n", "").lower()
-    return password
+
+    words = []
+    with open("/usr/share/dict/american-english", encoding="utf-8") as word_file:
+        words = [word.strip() for word in word_file if len(word.strip()) >= 4]
+
+    # Generate a passphrase with two random words and a random separator
+    word1 = secrets.choice(words).lower()
+    word2 = secrets.choice(words).lower()
+    separator = secrets.choice(["-", "_", ".", "~"])
+
+    # Add a random number and a special character for extra security
+    random_number = secrets.randbelow(100)
+    special_char = secrets.choice(string.punctuation)
```

Comment:
This is really not needed, if there is a concern about the password, it just just be increased to another word